pinot-common/src/main/java/org/apache/pinot/sql/parsers/CalciteSqlParser.java

```diff
@@ -610,6 +610,10 @@ private static Expression toExpression(SqlNode node) {
         return RequestUtils.getIdentifierExpression(node.toString());
       case LITERAL:
         return RequestUtils.getLiteralExpression((SqlLiteral) node);
+      case IS_NULL:
```

Comment:
The change in this class is not required as they can be handled the same as regular function (e.g. no special handling for `GREATER_THAN`)

Reply:
done